tests/unit/introspection.tcl

```diff
@@ -361,30 +363,31 @@ start_server {tags {"introspection"}} {
         foreach c [dict keys $some_configs] {
             lappend backups $c [lindex [r config get $c] 1]
         }
-        
 
         set used_port [expr ([dict get $backups port]+1)%65536]
         dict set some_configs port $used_port
 
-
         # Run a dummy server on used_port so we know we can't configure redis to 
         # use it. It's ok for this to fail because that means used_port is invalid 
         # anyway
-        catch {socket -server dummy_accept $used_port}
+        catch {socket -server dummy_accept -myaddr 127.0.0.1 $used_port} e
```

Comment:
don't know yet.. it took me some times to try it out.

It may be related to the binding to IPV6 in tcl socket by dafault?
btw, i tested it in one of Mac machine (tcl8.5 , M1), by default it bind to IPV4.

And the other Mac machine is in my house, the one that fail the test last night.
i will do the test again on another mac machine when i come home. (tcl8.6 (intel)) tonight